code.gs

```diff
@@ -1216,10 +1216,29 @@ function getEventsOnDate (eventDate, calendarId) {
  * of his/her contacts scheduled for the next days.
  */
 function main (forceDate) {
+  log.add('main() running.', 'info');
+  
+  var emailData = generateEmailNotification(forceDate);
+  
+  //if generateEmailNotification returned mail content send it
```

Comment:
[Javascript Semistandard style](https://github.com/GioBonvi/GoogleContactsEventsNotifier/blob/master/.github/CONTRIBUTING.md#coding-guidelines) requires a space between the `//` comment symbol and the first letter of the comment.